README.md

```diff
@@ -34,13 +34,11 @@ RoboHome is a SaaS tool that also integrates with Amazon's Echo to enable contro
 
 ### Docker :whale2:
 
-Docker support is currently on hold while the port to Laravel takes place.
-
 This project uses [Docker Compose](https://docs.docker.com/compose/) to help easily emulate a test environment for rapid development and testing.  This container has the PHP, MySQL, and phpMyAdmin services.
 
-There are two things different from setup manually:
-1. Copy config in `.env.docker` file into `.env` instead of using `.env.example`
-2. `composer` and `php artisan` are preinstalled commands in Docker container, you can use via `./web.sh` script. For example: `./web.sh composer install` or `./web.sh php artisan migrate`. This script would run in container automatically 
+There are a few configuration steps that ar different from the manual configuration:
+1. Rename `.env.docker` to `.env`
+2. `composer` and `php artisan` are preinstalled commands in Docker container that you can use via the `./web.sh` script in the root directory. For example: `./web.sh composer install` to install Composer dependencies on the container or `./web.sh php artisan migrate` to run the Laravel migrations. This script would run in container automatically 
```

Comment:
It looks like a few of my more subtle edits didn't make it in and there are some typos, please copy and paste the below to make sure we get them.  Markdown doesn't get copy and pasted, so make sure you add in backticks and numbers when necessary.

---

There are a few configuration steps that are different from the manual configuration:

1. Rename `.env.docker` to `.env`.
2. `composer` and `php artisan` are preinstalled commands in the Docker container that you can use via the `web.sh` script in the root directory.  For example: `./web.sh composer install` to install Composer dependencies on the container or `./web.sh php artisan migrate` to run the Laravel migrations on the container.  This script will run in container automatically.